docs/codespace_pr_reviews.md

```diff
@@ -0,0 +1,41 @@
+# Using Codesapces to review PRs without pulling a repo to your machine
+
+1. go to the PR
+2. click the `Code` button in the top right corner
+3. click the `Codespaces` tab
+4. `Create a new codespace on ...`
+
+   ![Create a new codespace screenshot](https://github.com/trussworks/react-uswds/assets/59394696/9ebfff8a-353c-4064-bf70-8712bbfbbeda)
+
+5. **Wait** for the codespace to finish setting up, you should see stuff happening in the `Terminal`. This is the codespace installig the correct versions of packages etc will run in the environment, so it's not necessary to have those updated locally, or even have the repo on your machine locally at all. If you mess with it before it is done it might not work. If you mess this up, you might see an error about your node version.
+
+   When it is complete, you should see a message like this in the terminal. If you don't see this, you should delete the codespace and start over.
+
+   ![Github Codespace configuaration process](https://github.com/trussworks/react-uswds/assets/59394696/bdf6cd8d-22e2-4e6b-b778-65c83c51556a)
+
+6. if the yarn install in that process was not successful, like in the image above, run `yarn install`. Wait until it is done before you do anything. takes ~5ish minutes
```

Comment:
I first read this as meaning "was not successful, like in the image [showing unsuccess] above".

```suggestion
6. if the yarn install in that process was not successful (should look like the image above), run `yarn install`. Wait until it is done before you do anything. takes ~5ish minutes
```